### ISSUE TITLE
Could you support setTemplate page by page form the external document?

### ISSUE THREAD

**Issue by secreter** [rkusa/pdfjs#274]
I have a external PDF document , it has 10 pages.

**Comment by rkusa** [comment]
You are right, this cannot be achieved with `pdfjs` right now. While there might be a workaround by adding each page as a _background image_, I wouldn't recommend going that route.
